lab-anthony/lib/greet.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
```

Comment:
There should be three files for for this submission. 
* One file should only contain the greet function, and add it as the greet property of the exports object
*  another should require the greet function and run it (that will be your command line program)
* a third file in the test directory, that should test the greet function